src/views/home/subviews/HomeTripSubview.tsx

```
@@ -0,0 +1,393 @@
+            duration: 15,
+            description: `Bus line 8`,
+            busLine: "8",
+            fromStop: "Transfer point",
+            toStop: Stops[toStopId]?.[3] || "Stop",
+          },
+        ],
+      },
+    ];
+  };
+
+  const routes = useMemo(generateMockRoutes, [fromStopId, toStopId]);
```

ESLint

[error] Error: Expected the first argument to be an inline function expression

Expected the first argument to be an inline function expression.

/home/jailuser/git/src/views/home/subviews/HomeTripSubview.tsx:165:26
  163 |   };
  164 |
> 165 |   const routes = useMemo(generateMockRoutes, [fromStopId, toStopId]);
      |                          ^^^^^^^^^^^^^^^^^^ Expected the first argument to be an inline function expression
  166 |
  167 |   const selectFromStop = (stopId: number): void => {
  168 |     setFromStopId(stopId);

(react-hooks/use-memo)